applications/application_life_cycle_management/odc-working-with-helm-charts-using-developer-perspective.adoc

```diff
@@ -8,6 +8,17 @@ toc::[]
 include::modules/helm-understanding-helm.adoc[leveloffset=+1]
 
 
-You can use the xref:../../web_console/odc-about-developer-perspective.adoc#odc-about-developer-perspective[*Developer* perspective] in the xref:../../web_console/web-console.adoc#web-console[web console] to create Helm releases.
+You can use the *Developer* perspective in the web console to select and install a chart from the Helm charts  listed in the *Developer Catalog*. You can create a Helm release using these charts, upgrade, rollback, and uninstall the release.
```

Comment:
is there an extra space between "charts" and "listed"?

Reply:
Fixed